data-esnext.js

```diff
@@ -1179,6 +1179,7 @@ exports.tests = [
         babel: true,
         typescript: typescript.corejs,
         duktape20: false,
+        chrome60: true,
```

Comment:
Yes I'm pretty sure.
<img width="685" alt="screen shot 2017-04-21 at 10 46 53" src="https://cloud.githubusercontent.com/assets/1507086/25270165/e7288860-267f-11e7-9f56-4153846431aa.png">

Canary has A/B testing as was discussed before in older issues and PRs so maybe you are just the one of those lucky testers :)